book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,234 @@
+
+(rr-rdm-elns-choosing-compare-to-paper)=
+### Always compare to paper notebooks
+
+When picking an ELN remember to always compare the ELN to a paper notebook not just to other ELNs, it's likely paper that you are replacing so it is the benchmark against which to compare.
+Consider the differences between a paper copy of a lab notebook and an electronic copy.
+Consider also which are the properties of a paper copy that it is important that you are able to retain when adopting an electronic alternative.
+
+-   A paper lab notebook is physically under your control.
+-   You (or more likely your organisation) own it.
+-   You can control access to it physically.
+-   Your physical possession of this resource means that it would be difficult for anyone to prevent you from accessing it.
+-   Conversely it must be physically stolen to be accessed by a malicious third party.
+-   On the other hand, ELNs - like all software - have security vulnerabilities.
+-   You are not dependent on the functioning of any complex systems like computer networks in order to be able to use your paper lab notebook.
+-   You do not need any specialist tools in order to access its contents.
+-   There is no reason why you might have to pay a 3rd party a fee in order to continue using it.
+-   You do not have to agree to a 'terms of service' or 'end-user license agreement' with a third party, the terms of which may be subject to unilateral alteration by that third party, in order to use and retain access to your lab book.
+
+If your provider of paper lab books goes out of business it has almost no bearing on your ability to continue doing your work.
+One paper notebook is much like another, finding a new provider is pretty easy.
+Changing providers does not impact on your ability to access your past notebooks or to continue operating with the same workflow in future ones.
+This is not necessarily true of ELNs.
+
+(rr-rdm-elns-choosing-archival)=
+### Archival function of notebooks
+
+Few active measures are needed to maintain the data in your paper notebooks, they are vulnerable as they exist in only one copy. 
+Nevertheless, as long as they are kept in a cool, dry and dark spot for example a fireproof safe they will likely last decades.
+Electronic data requires much more active upkeep.
+Electronic data can *if manged properly* be more resilient to physical threats such as fire and flooding as it can frequently be backed up in multiple locations.
+A hard drive or even a solid state drive however cannot be left in a draw for a decade or more and have a high likelihood of working without some amount of bitrot or compatability issue when plugged back in.
+
+Lab notebooks perform an archival function and proprietary formats are antithetical to this as they assume the institution which can act as a gatekeeper to the use of the proprietary format will outlive the need to archive the material.
```

Comment:
```suggestion
Lab notebooks perform an archival function.
Proprietary formats are antithetical to this as this assumes that the proprietary format will remain functional for the archival period. 
```
Or something else to clarify this sentence

Reply:
clarified this here [c567a05](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/c567a051c56e793020f50c6e73f8e7a450633286), &
[c7b38f0](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/c7b38f07b4f03e7c310da34e60c3b2f6ac167647) hopefully not too long now